joomla/joomla-cms
# Add path filter to joomla updates download process for the temp folder

### Summary of Changes

The JSST has been contacted about an missing path filter whithin the joomla download process. Given that an successfull attack requires a Super User access to change the tmp path setting and also to trigger the update itself the JSST decided to move this patch to the public tracker.

### Testing Instructions

- apply the patch
- Try to install an update or to "reinstall the core files"
- Make sure the update works as expected
- try to install an update via this custom update server (https://ci.joomla.org/artifacts/joomla/joomla-cms/staging/32076/downloads/39782/pr_list.xml) 

### Actual result BEFORE applying this Pull Request

The upgrade works as expected

### Expected result AFTER applying this Pull Request

The upgrade works as expected

### Documentation Changes Required

none
